### PR TITLE
Increment Key Vault versions after the May 2024 release.

### PR DESCRIPTION
**Pull request by vcolin7**

### CODE REVIEW

File: sdk/clientcore/http-stress/pom.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-monitor-opentelemetry-exporter</artifactId>
-      <version>1.0.0-beta.21</version> <!-- {x-version-update;com.azure:azure-monitor-opentelemetry-exporter;dependency} -->
+      <version>1.0.0-beta.22</version> <!-- {x-version-update;com.azure:azure-monitor-opentelemetry-exporter;dependency} -->
```
Comment (mssfang) [inline sdk/clientcore/http-stress/pom.xml:58]: The changes seem not related to Key Vault

**Review by anuchandy — APPROVED** [review]

**Review by mssfang — APPROVED** [review]

**Review by conniey — APPROVED** [review]

File: sdk/clientcore/http-stress/pom.xml (hunk 1 above)
Comment (vcolin7) [reply sdk/clientcore/http-stress/pom.xml:58]: Apparently, we hadn't updated all the dependencies our stress test framework uses in the new generic core directory. it's fine that we update that here.
